src/main/java/io/jenkins/plugins/thememanager/ThemeManagerFactory.java

```diff
@@ -43,7 +43,7 @@ public String getCssUrl() {
    */
   public String toAssetUrl(String asset) {
     ThemeManagerFactoryDescriptor descriptor = getDescriptor();
-    return Jenkins.get().getRootUrlFromRequest() + "theme-" + descriptor.getThemeId() + "/" + asset;
+    return Jenkins.get().getRootUrl() + "theme-" + descriptor.getThemeId() + "/" + asset;
```

Comment:
it was changed here:
https://github.com/jenkinsci/theme-manager-plugin/issues/27

Generally this means you haven't configured Jenkins to be aware of your reverse proxy,

The x-forwarded headers need to be set:
X-Forwarded-Proto, X-Forwarded-Host, X-Forwarded-Port

Reply:
Ok, thank you for your information.

But my opinion is that Jenkins URL configuration should take precedence over the headers. Specially in big corporations is not always easy to have the headers properly configured. 

All other places Jenkins use the Jenkins URL configuration, but to load this CSS.

Reply:
@daniel-beck any opinion?

Reply:
For URL references during request handling (rather than notification emails at the end of the build), the current code is correct.